.github/workflows/publish-to-pypi-test.yml

```diff
@@ -113,25 +117,54 @@ jobs:
         with:
           password: ${{ secrets.TEST_PYPI_API_TOKEN }}
           repository_url: https://test.pypi.org/legacy/
-        if: github.ref == 'refs/heads/master'
+        if: ${{ (github.ref == 'refs/heads/master' && ! contains(toJSON(github.event.commits.*.message), 'Bump version:')) }}
+        
 
   release:
-    runs-on: ubuntu-latest
+    name: Release build on git
+    runs-on: ubuntu-18.04
     needs: [build-n-publish]
-    if: github.ref == 'refs/heads/master'
+    if: ${{ (github.ref == 'refs/heads/master') }}
+
     steps:
-    - uses: actions/checkout@master
-    - name: Create release
-      uses: Roang-zero1/github-create-release-action@master
-      with:
-        version_regex: ^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - name: Create GitHub release
-      uses: Roang-zero1/github-upload-release-artifacts-action@master
-      with:
-        args: '[
-        , ./dist/*
-        ]'
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }} 
+          fetch-depth: 0
+        
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      
+      - name: Install pypa/build
+        run: |
+          python -m pip install build --user
+
+      - name: Assign Git User Email
+        run: |
+          git config --global user.email "mrbump@nebra.com"
```

Comment:
Surprised we need to do this config, don't see anything similar in hm-miner: https://github.com/NebraLtd/hm-miner/blob/master/.github/workflows/check-latest-release.yml

Reply:
This is for the commit that Mr. Bump is doing, it gave me an error the first time I implemented him saying unknown user and to set the name + email